proxy/pom.xml

```diff
@@ -598,6 +598,12 @@
       <artifactId>snakeyaml</artifactId>
       <version>2.0</version>
     </dependency>
+    <dependency>
+      <groupId>org.projectlombok</groupId>
+      <artifactId>lombok</artifactId>
+      <version>RELEASE</version>
+      <scope>compile</scope>
+    </dependency>
```

Comment:
Please, remove this, and the need to gets methods.
We already have too many dependencies, and this jus save you few very simples lines of code.

Reply:
Done